framework/src/mysql/mysql.js

```diff
@@ -341,28 +341,44 @@ const getTableInstance = async (...tableParams) => {
 			params.search = Array.isArray(params.search) ? params.search : [params.search];
 
 			params.search.forEach(search => {
-				const { property, pattern, startsWith, endsWith } = search;
-				if (pattern) query.where(`${property}`, 'like', `%${pattern}%`);
-				if (startsWith) query.where(`${property}`, 'like', `${startsWith}%`);
-				if (endsWith) query.where(`${property}`, 'like', `%${endsWith}`);
+				const { property, pattern, startsWith, endsWith, allowWildCards } = search;
+				if (allowWildCards === true) {
+					if (pattern) query.where(`${property}`, 'like', `%${pattern}%`);
+					if (startsWith) query.where(`${property}`, 'like', `${startsWith}%`);
+					if (endsWith) query.where(`${property}`, 'like', `%${endsWith}`);
+				} else {
+					if (pattern) query.where(`${property}`, 'like', `%${pattern.replace(/%/g, '\\%').replace(/_/g, '\\_')}%`);
+					if (startsWith) query.where(`${property}`, 'like', `${startsWith.replace(/%/g, '\\%').replace(/_/g, '\\_')}%`);
+					if (endsWith) query.where(`${property}`, 'like', `%${endsWith.replace(/%/g, '\\%').replace(/_/g, '\\_')}`);
+				}
 			});
 		}
 
 		if (params.orSearch) {
 			params.orSearch = Array.isArray(params.orSearch) ? params.orSearch : [params.orSearch];
 
-			query.andWhere(function () {
+			query.andWhere(() => {
```

Comment:
Arrow functions don't have the `this` keyword available. Can we please fallback to using the standard `function` declaration instead?